src/components/pacbio/PacbioRunWellEdit.vue

```diff
@@ -215,23 +240,25 @@ export default {
         this.currentWell.ccs_analysis_output = 'No'
       }
     },
-    updateAdaptiveLoading() {
-      if (this.currentWell.loading_target_p1_plus_p2) {
-        this.currentWell.use_adaptive_loading = 'True'
-      } else {
-        this.currentWell.use_adaptive_loading = 'False'
-      }
-    },
     formatLoadingTargetValue(val) {
       if (val) {
         if (this.decimalPercentageRegex.test(val)) {
           return val
         } else {
-          this.loadingTargetValue = parseFloat(val / 100).toFixed(2)
           return isNaN(this.loadingTargetValue) ? 0 : this.loadingTargetValue
         }
       }
     },
+    disableAdaptiveLoadingInput() {
+      this.disableAdaptiveLoading = true
+      this.currentWell.loading_target_p1_plus_p2 = ''
+      this.use_adaptive_loading = 'False'
```

Comment:
Is this attribute still relevant? I thought disableAdaptiveLoading replaced use_adaptive_loading?

Reply:
I am using this to set the 'disabled' attribute of the input box to true/false

Reply:
OK. You could follow James' comments above about creating a computed field then you would not need to set the value.

```
{ // in computed
  useAdaptiveLoading: (component) => component.currentWell.loading_target_p1_plus_p2 !== '' 
}
```